ocs_ci/ocs/ui/base_ui.py

```diff
@@ -324,7 +325,7 @@ def choose_expanded_mode(self, mode, locator):
         locator (tuple): (GUI element needs to operate on (str), type (By))
 
         """
-        current_mode = self.is_expanded(locator=locator, timeout=1000)
+        current_mode = self.is_expanded(locator=locator, timeout=180)
```

Comment:
there is no case we wait 16 minutes for UI to be updated after click. It is just a navigation btn + frontend script. That should happen normally less than 2 sec, otherwise we can open a bug. So wait for 16 min is just wasting resources and blocking other tests to run; I just accidentally found it.